Makefile

```diff
@@ -263,19 +263,32 @@ check-contrib:
 	@echo Setting contrib at $(CONTRIB_PATH) to use this core checkout
 	@$(MAKE) -C $(CONTRIB_PATH) for-all CMD="$(GOCMD) mod edit \
 		$(addprefix -replace ,$(join $(ALL_MOD_PATHS:%=go.opentelemetry.io/collector%=),$(ALL_MOD_PATHS:%=$(CURDIR)%)))"
-	@$(MAKE) -C $(CONTRIB_PATH) gotidy
+	@$(MAKE) -C $(CONTRIB_PATH) for-all CMD="$(GOCMD) mod tidy"
+
+	@if [ -z "$(SKIP_REGENERATE)" ]; then \
+		$(MAKE) regenerate-contrib; \
+	fi
```

Comment:
if you're running `check-contrib` locally, know you haven't modified mdatagen, and want to speed things up a bit, i guess? not sure how common a use case it would be.

Reply:
Unless you have found it useful yourself, I would rather keep it simple for now until somebody asks for it

Reply:
alright, i'll remove it.